nvflare/private/fed/app/server/server_train.py

```diff
@@ -158,5 +160,7 @@ def parse_arguments():
 
     version_check()
     args = parse_arguments()
+    if args.start_method and args.start_method in ["spawn", "fork", "forkserver"]:
+        multiprocessing.set_start_method(args.start_method)
```

Comment:
This may not work. The GPU subprocess is started from child process and this needs to set in the child process.